src/dodal/devices/areadetector/adsim.py

```diff
@@ -1,47 +1,45 @@
-from ophyd.areadetector.base import ADComponent as Cpt
-from ophyd.areadetector.detectors import DetectorBase
+from bluesky.protocols import HasHints, Hints
+from ophyd_async.core import (
+    DirectoryProvider,
+    StandardDetector,
+)
+from ophyd_async.epics.areadetector.controllers import ADSimController
+from ophyd_async.epics.areadetector.drivers import ADBase, ADBaseShapeProvider
+from ophyd_async.epics.areadetector.writers import HDFWriter, NDFileHDF, NDPluginStats
 
-from .adutils import Hdf5Writer, SingleTriggerV33, SynchronisedAdDriverBase
 
+class AdSimDetector(StandardDetector, HasHints):
```

Comment:
Already defined in ophyd-async

https://github.com/bluesky/ophyd-async/blob/e4fb8492bfad29dff9f539f6adc63c8152f68493/src/ophyd_async/epics/adsimdetector/_sim.py#L9

Reply:
https://github.com/bluesky/ophyd-async/blob/ce3b85ded7811843d7c984a4911c5799ff05eb55/src/ophyd_async/epics/demo/demo_ad_sim_detector.py#L10

This is the only place sim_detector is present and it is not exported 

Should I increase the version of ophyd_async but it will be a big change

Reply:
Ah, I didn't realise that we hadn't updated to ophyd-async yet. Let's find out who's doing that.